### ISSUE TITLE
Error when run get_receptor_inference

### ISSUE THREAD

**Issue by SejeongPark8354** [HannesStark/EquiBind#20]
First of all, Your research has had a huge impact on drug discovery based on AI. Thank you so much!!!

I got the error like below. 
My Inputs are protein pdb from https://www.rcsb.org/ and 3d ligand conformer from PubChem. 

```
[2022-04-06 19:21:16.672575] [ Using Seed :  1  ]

Processing SOS1: complex 1 of 1
Trying to load data/my_data_folder/SOS1/SOS1_ligand.sdf
Docking the receptor data/my_data_folder/SOS1/SOS1_protein.pdb
To the ligand data/my_data_folder/SOS1/SOS1_ligand.sdf
Traceback (most recent call last):
  File "inference.py", line 473, in <module>
    inference_from_files(args)
  File "inference.py", line 340, in inference_from_files
    rec, rec_coords, c_alpha_coords, n_coords, c_coords = get_receptor_inference(rec_path)
  File "/home/sejeong/codes/EquiBind/commons/process_mols.py", line 421, in get_receptor_inference
    c_alpha_coords = np.concatenate(valid_c_alpha_coords, axis=0)  # [n_residues, 3]
  File "<__array_function__ internals>", line 6, in concatenate
ValueError: need at least one array to concatenate
```

Is there any problem in my input setting? 
I would be very grateful if you could give me a solution. 

Thank you for your work again. :)

**Comment by nohandsomewujun** [comment]
Have you solved the problem yet, I have the same problem as you。😂

**Comment by SejeongPark8354** [comment]
@nohandsomewujun 
I haven't solved this problem yet 😭

**Comment by HannesStark** [comment]
Sorry for forgetting about this and thanks for the reminder!

I am not sure why the array is empty in your case.
Could you send me (hstark@mit.edu) the pdb file and the molecule file for which this is happening so I can try to reproduce it?

**Comment by HannesStark** [comment]
Soo, @nohandsomewujun sent me his files yesterday and I was able to reproduce and solve the problem.
If you pull the latest update it should work - thanks for pointing out this issue!

The issues:
I assumed that waters (HOH) were in separate chains in pdb files and these chains were removed. In the pdb files of @nohandsomewujun, every backbone chain also contained water residues and all chains were removed.

P.S.
@nohandsomewujun, I noticed that your ligand file was missing hydrogens. Make sure to add them before running EquiBind!
